source/getting-started/presence-detection.markdown

```diff
@@ -15,12 +15,16 @@ Screenshot of Home Assistant showing a school, work and home zone and two people
 
 ### Adding presence detection
 
-There are different ways of setting up presence detection. Usually the easiest way to detect presence is by checking which devices are connected to the network. You can do that if you have one of our [supported routers][routers]. By leveraging what your router already knows, you can easily detect if people are at home.
+There are different ways of setting up presence detection using the official intergrations. Usually the easiest way to detect presence is by checking which devices are connected to the network. You can do that if you have one of our [supported routers][routers] which will provide information on which devices are connected. By leveraging what your router already knows, you can easily detect if people are at home. One thing to consider is that some devices drop of the network to save battery, which could lead to false state changes.
```

Comment:
- `using the official integrations` -> What does that mean? Why is that added?
- `One thing to consider is that some devices drop of the network to save battery, which could lead to false state changes.` -> Not sure if a new users understands this line.

Considering the goal here, I'm not sure if we should talk about routers at all. Those devices tend to spawn a lot of entities that only add more confusion.